stdlib/cache/src/main/ballerina/src/cache/cache_errors.bal

```diff
@@ -47,3 +47,19 @@ function prepareError(string message, error? err = ()) returns Error {
     }
     return cacheError;
 }
+
+# Prepare the `error` as an `Error`.
+#
+# + message - Error message
+# + err - `error` instance
+# + return - Prepared `Error` instance
+function prepareErrorWithDebugLog(string message, error? err = ()) returns Error {
+    log:printDebug(message, err);
```

Comment:
Don't we need to check whether the debug has enabled or not?

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/21774/commits/d318ea7f0a283342354a9095c6d44c18fdbc1753